### PR TITLE
Sorting seed phrase confirmation buttons alphabetically

### LINKED ISSUE #7799: "Confirm your Secret Backup Phrase" should be sorted.
Originally posted over at 
Migrated from https://github.com/brave/brave-browser/issues/7630, which indicated this was actually a MetaMask bug as they just run a MetaMask fork.

## Description 
When creating a new Brave Wallet, it gives me a seed phrase to note and then asks me to confirm it by selecting words from a list in the order they appeared in the seed phrase.  The problem is the list of words is unordered, so when I try to find the next word in my seed phrase I have to painstakingly search the word list one word at a time.  The list should be sorted in alphabetical order so it is easy to find the next word via a mental binary search (the way humans naturally try to search sorted lists).

## Steps to Reproduce
   1. Settings > Crypto Wallets > New Wallet
   2. Write down seed.
   3. Try to confirm.

## Actual result:
Pain and suffering that make me want to not use the feature.

## Expected result:
Delight at using modern financial technology.

## Reproduces how often: 
100%.

## Brave version (brave://version info)
Brave | 1.2.41 Chromium: 79.0.3945.88 (Official Build) (64-bit)
-- | --
Revision | c2a58a36b9411c80829b4b154bfcab97e581f1f3-refs/branch-heads/3945@{#954}
OS | Windows 10 OS Version 1903 (Build 18362.535)

## Version/Channel Information:
- Can you reproduce this issue with the current release? Yes.
- Can you reproduce this issue with the beta channel? Unknown
- Can you reproduce this issue with the dev channel? Unknown
- Can you reproduce this issue with the nightly channel? Unknown

### PR DESCRIPTION
**Pull request by ryanml**
Fixes: https://github.com/MetaMask/metamask-extension/issues/7799

Screen Shot 2020-01-29 at 11 36 56 AM

`sort` does the trick fine so I removed the `lodash` inclusion from this component

### CODE REVIEW

File: ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js [hunk 1]
```diff
@@ -42,7 +41,7 @@ export default class ConfirmSeedPhrase extends PureComponent {
 
   componentDidMount () {
     const { seedPhrase = '' } = this.props
-    const shuffledSeedWords = shuffle(seedPhrase.split(' ')) || []
+    const shuffledSeedWords = (seedPhrase.split(' ') || []).sort()
```
Comment (whymarrh) [inline ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js]: We might want to rename this to `sortedSeedWords` or similar?

**Review by whymarrh — CHANGES_REQUESTED** [review]

File: ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js (hunk 1 above)
Comment (ryanml) [reply ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js]: Ah, indeed, let me update :)

Comment (ryanml) [reply ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js]: done

Comment (Gudahtt) [reply ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js]: Thanks! Though there are a couple of instances left of a variable called `shuffledIndex` as well it looks like

Comment (ryanml) [reply ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js]: @Gudahtt let me get those and fix tests

Comment (ryanml) [reply ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js]: @Gudahtt done, I've also updated css classnames

Comment (Gudahtt) [reply ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js]: Ah right, good catch! Forgot about those

**Review by Gudahtt — APPROVED** [review]
Thanks, looks good to me!

**Review by whymarrh — APPROVED** [review]
LGTM, thanks @ryanml!

**Pull request merged**